.env.example

```
@@ -4,15 +4,14 @@ GITHUB_WEBHOOK_SHARED_SECRET=
 OVH_APP_KEY=
 OVH_APP_SECRET=
 OVH_CONSUMER_KEY=
-SESSION_SECRET=
+SESSION_SECRET=LOCALHOST
```

c'est ambigu de mettre "LOCALHOST", un truc comme "changeme" serait mieux. Voir même une erreur pour forcer l'utilisateur à mettre une bonne valeur. (on peut aussi commenté la ligne)

---

```suggestion
SESSION_SECRET=ChangeIdDoNoLeetADefaultValueInProduction
```
J'hésite à la mettre en commenté aussi

---

comme tu le sens, je peux l'enlever, avec docker-compose on a une variable d'environnement qui ne posera pas de problème si on laisse ce champs vide